### DISCUSSION TITLE
Multiple options support

### DISCUSSION THREAD

**Discussion by tanner0101** [spectreconsole/spectre.console#465] [Q&A]
I'd like to be able to support multiple values for an option in my CLI tool. For example, something similar to Helm's `--values` or `--set` flags:

```
 helm upgrade --values myvalues.yaml --values override.yaml redis .
```

Here `--values` is equal to the array `["myvalues.yaml", "override.yaml"]`. 

I couldn't find a way to do this, so I think it's not supported. I could imagine declaring such an option like this:

```cs
[CommandOption("--values")]
public List<string> Values { get; set; }
```

Thoughts on supporting this? I'd be happy to make a PR if so!

**Comment by patriksvensson** [comment]
@tanner0101 This should already be possible unless I misunderstand your question:

```
[CommandOption("--values <VALUES>")]
public string[] Values { get; set; }
```

**Reply by patriksvensson** [reply]
*Migrated this to Q/A discussion*
